### ISSUE TITLE
Add Demo: Simple Image Classification Demo (e.g. MNIST) with Convolutional Layers

### ISSUE THREAD

**Issue by ltiao** [data61/aboleth#75]
Demo should output a plot like this:

![mc_dropout_mnist_prob](https://user-images.githubusercontent.com/1112238/28951037-c6050a76-790b-11e7-85c9-be3a9c8c274f.jpg) from [What My Deep Model Doesn't Know... (Yarin Gal)](http://mlg.eng.cam.ac.uk/yarin/blog_3d801aa532c1ce.html).

Anyone can obtain ~99% accuracy results on MNIST with any old deep learning framework, but Aboleth allows us to obtain a distribution over the logits and softmax outputs *easily*, by independently taking samples of ConvNets and making forward passes through each.
